dss/api/bundles/__init__.py

```diff
@@ -65,8 +67,29 @@ def get(
             headers['Retry-After'] = RETRY_AFTER_INTERVAL
             return response
 
+    all_files = bundle_metadata[BundleMetadata.FILES]
+
+    link = None
+    if len(all_files) - start_at > per_page:
+        next_url = (UrlBuilder().set(path=f"v1/bundles/{uuid}")
+                    .add_query("replica", _replica.name)
+                    .add_query("version", version)
+                    .add_query("per_page", str(per_page))
+                    .add_query("start_at", str(start_at + per_page)))
+        if directurls:
+            next_url.add_query("directurls", "true")
+        if version is not None:
```

Comment:
Doesn't Line:76 to this already and is version ever none?

Reply:
fixed